test/util/event_stream_spec.js

```diff
@@ -63,15 +63,15 @@ describe('EventStream', function() {
     stream._schedule.verify();
   });
 
-  it('should end on error if not handled', function() {
-    var stream = new EventStream(events, 123);
-    var callback = readForCatch(stream);
-    stream._schedule = sinon.mock().never();
-    assert.throws(function() {
-      callback('fake_error');
-    }, 'fake_error');
-    stream._schedule.verify();
-  });
+//  it('should end on error if not handled', function() {
```

Comment:
Good find! Generaly, whenever code is commented out, it should always be accompanied by a comment explaining why it's commented out. So it would be appropriate to explain here that this test breaks coverage reporting for some reason.